### PR TITLE
NH-46957: add notice

### PR DESCRIPTION
**Pull request by xuan-cao-swi**
For https://swicloud.atlassian.net/browse/NH-46957

Note the repository name for new `solarwinds-apm` v6+ is not yet finalized.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# Notice
+As of 2023-07-21, we regret to announce that **solarwinds-apm-ruby-gen-1** is not in active development. We have shift our focus on opentelemetry-based apm library. The new repository is located in https://github.com/solarwindscloud/solarwinds-apm-ruby
```
Comment (cheempz) [inline README.md:2]:
```suggestion
This is the renamed repository for `solarwinds-apm` versions below 6, which is now deprecated in favour of the OpenTelemetry-based `solarwinds-apm` starting from version 6, located at https://github.com/solarwindscloud/solarwinds-apm-ruby.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+
+Please be aware of the following implications:
+
+1. **Limited Updates**: There will be limited new feature updates, bug fixes, or enhancements to the **solarwinds-apm-ruby-gen-1** from our official development team.
+
+2. **Limited Maintenance**: We will still maintain the repository or address any issues or pull requests submitted, but the duration may take longer.
+
```
Comment (cheempz) [inline README.md:9]:
```suggestion
Please note this codebase is no longer in active development as we focus on enhancing `solarwinds_apm` version 6+.  Version 5 releases will be limited to critical updates and fixes, and issues or pull requests submitted to this repository may have a longer response time.
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+# Notice
+As of 2023-07-21, we regret to announce that **solarwinds-apm-ruby-gen-1** is not in active development. We have shift our focus on opentelemetry-based apm library. The new repository is located in https://github.com/solarwindscloud/solarwinds-apm-ruby
+
+Please be aware of the following implications:
```
Comment (cheempz) [inline README.md:4]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
+
+1. **Limited Updates**: There will be limited new feature updates, bug fixes, or enhancements to the **solarwinds-apm-ruby-gen-1** from our official development team.
```
Comment (cheempz) [inline README.md:6]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,12 @@
...
+
+2. **Limited Maintenance**: We will still maintain the repository or address any issues or pull requests submitted, but the duration may take longer.
```
Comment (cheempz) [inline README.md:8]:
```suggestion
```

**Comment by cheempz** [comment]
LOL I've epic failed at making the suggestions for multiple paragraphs.  @xuan-cao-swi I hope you don't mind but i'll just commit things to make it easier to read.

**Comment by cheempz** [comment]
@xuan-cao-swi did you want to fix the failing tests as part of a different PR?

**Comment by xuan-cao-swi** [comment]
> @xuan-cao-swi did you want to fix the failing tests as part of a different PR?
Hi Lin, I set the mariadb to last version.

**Review by cheempz — DISMISSED** [review]
Thanks for fixing the tests

**Comment by cheempz** [comment]
@xuan-cao-swi I reverted the changes about repository rename (see rationale in Jira ticket) so this PR would simply be adding the notice... and unfortunately looking at why tests failed 😢

**Comment by xuan-cao-swi** [comment]
> @xuan-cao-swi I reverted the changes about repository rename (see rationale in Jira ticket) so this PR would simply be adding the notice... and unfortunately looking at why tests failed 😢

Yeah, the new psych is causing problem. We need to update the dockerfile that install libyaml...

**Review by cheempz — APPROVED** [review]
👍 thanks @xuan-cao-swi!

**Pull request merged**